config.yaml

```diff
@@ -0,0 +1,142 @@
+          - Miscellanea:
+              location: content/chapters/functions/lecture
+              name: content/miscellanea
+        - Multivariate to Power:
+          - Overview:
+              location: content/chapters/multivariate-to-power/lecture
+              name: content/overview
+          - Multivariate probability distributions:
+              location: content/chapters/multivariate-to-power/lecture
+              name: content/multivariate-prob-distributions
+          - Some distributions related to the normal:
+              location: content/chapters/multivariate-to-power/lecture
+              name: content/distributions-normal
+          - Estimation, estimates and estimators:
+              location: content/chapters/multivariate-to-power/lecture
+              name: content/estimation
+          - Test of hypothesis, P values and related concepts:
+              location: content/chapters/multivariate-to-power/lecture
+              name: content/test-hypothesis-p-values
+          - Power and sample sizes:
+              location: content/chapters/multivariate-to-power/lecture
+              name: content/power-sample-sizes
+        - Vectors to Some Regression Topics:
+          - Overview:
+              location: content/chapters/vectors-matrix-ops/lecture
+              name: content/overview
+          - Vectors and Matrix Operations:
+              location: content/chapters/vectors-matrix-ops/lecture
+              name: content/vectors-matrix-operations
+          - Some notes on matrices and linear operators:
+              location: content/chapters/vectors-matrix-ops/lecture
+              name: content/matrices-linear-operators
+          - Ranks and determinants:
+              location: content/chapters/vectors-matrix-ops/lecture
+              name: content/ranks-determinants
+          - Multivariate calculus:
+              location: content/chapters/vectors-matrix-ops/lecture
+              name: content/multivariate-calculus
+          - The multivariate normal distribution and related topics:
+              location: content/chapters/vectors-matrix-ops/lecture
+              name: content/multivariate-normal-distribution
+          - Independence, expectations and the moment generating function:
+              location: content/chapters/vectors-matrix-ops/lecture
+              name: content/independence
+          - The gamma distribution:
+              location: content/chapters/vectors-matrix-ops/lecture
+              name: content/gamma-distribution
+          - Notes and examples - The linear model:
+              location: content/chapters/vectors-matrix-ops/lecture
+              name: content/examples-linear-model
+          - Some regression topics:
+              location: content/chapters/vectors-matrix-ops/lecture
+              name: content/regression-topics
+    config_meta:
+      title: Computing and Calculus for Applied Statistics
+      url: http://localhost/
+      baseUrl: /
```

Comment:
I tried setting this to `/ccas/`, but the resulting website throws an error and says that the `baseUrl` is wrong and that I should set it to `/`.